src/test/kotlin/com/cjbooms/fabrikt/generators/SpringControllerGeneratorTest.kt

```diff
@@ -140,6 +139,21 @@ class SpringControllerGeneratorTest {
         ).isTrue()
     }
 
+    @Test
+    fun `ensure controller methods has the correct method names`() {
```

Comment:
Looks good to me, but I would say just piggy-back on one of the existing tests instead of duplicating the resources